opentreemap/treemap/js/src/mapPage/browseTreesMode.js

```diff
@@ -49,11 +52,19 @@ function init(options) {
                 .filter(inMyMode),
 
         popupHtmlStream = utfEventStream.flatMap(getPopupContent),
+
         clickedIdStream = utfEventStream.map('.data.' + config.utfGrid.mapfeatureIdKey),
-        accordionHtmlStream = BU.fetchFromIdStream(clickedIdStream,
+        pagingIdStream = $('body').asEventStream('click', dom.popupPagingButtons)
+            .doAction(changePopupSelection)
+            .doAction(toggleAccordion, '')
+            .debounce(300)
+            .map(getIdFromPopup),
+        idStream = Bacon.mergeAll(clickedIdStream, pagingIdStream),
+
+        accordionHtmlStream = BU.fetchFromIdStream(idStream,
```

Comment:
I ran into a strange bug when trying to use quick edit to "unstack" trees on top of each other.

- Create an instance.
- Add a tree with a species.
- Add a tree with a different species directly on top of the first.
- Refresh the map page.
- Select the tree stack and navigate to the second item in the popup.
- Use Quick Edit to move the tree further than 20ft away.
- Save and refresh the page

Expected: The popup for each dot shows details for a single tree
Actual: The popup for the original dot shows both items. The popup for the moved dot shows just the one item. The plot count does not increase, so it does not look like things are being duplicated.